tests/variable_parse/valid0_new.env

```diff
@@ -0,0 +1,15 @@
+declare -- BASH="/bin/bash"
+declare -r BASHOPTS="checkwinsize:cmdhist:complete_fullquote:expand_aliases:extglob:extquote:force_fignore:globasciiranges:globskipdots:histappend:interactive_comments:login_shell:patsub_replacement:progcomp:promptvars:sourcepath"
+declare -i BASHPID
+declare -i BASHPID2="2"
+declare -a BASH_ALIASES=()
+declare -a BASH_ARGC=([0]="0")
+declare -ar BASH_VERSINFO=([0]="5" [1]="2" [2]="21" [3]="1" [4]="release" [5]="x86_64-pc-linux-gnu")
+declare -- COMP_WORDBREAKS=$' \t\n"\'><=;|&(:'
+declare -- IFS=$' \t\n'
```

Comment:
Is this the only relevant escaped variable?

Reply:
There are a few more (git related ones) that use ANSI-C string escapes on my system:

```
declare -- COMP_WORDBREAKS=$' \t\n"\'><=;|&(:'
declare -- IFS=$' \t\n'
declare -- __git_color_moved_ws_opts=$'no ignore-space-at-eol ignore-space-change\n\t\t\tignore-all-space allow-indentation-change'
declare -- __git_diff_common_options=$'--stat --numstat --shortstat --summary\n\t\t\t--patch-with-stat --name-only --name-status --color\n\t\t\t--no-color --color-words --no-renames --check\n\t\t\t--color-moved --color-moved= --no-color-moved\n\t\t\t--color-moved-ws= --no-color-moved-ws\n\t\t\t--full-index --binary --abbrev --diff-filter=\n\t\t\t--find-copies --find-object --find-renames\n\t\t\t--no-relative --relative\n\t\t\t--find-copies-harder --ignore-cr-at-eol\n\t\t\t--text --ignore-space-at-eol --ignore-space-change\n\t\t\t--ignore-all-space --ignore-blank-lines --exit-code\n\t\t\t--quiet --ext-diff --no-ext-diff --unified=\n\t\t\t--no-prefix --src-prefix= --dst-prefix=\n\t\t\t--inter-hunk-context= --function-context\n\t\t\t--patience --histogram --minimal\n\t\t\t--raw --word-diff --word-diff-regex=\n\t\t\t--dirstat --dirstat= --dirstat-by-file\n\t\t\t--dirstat-by-file= --cumulative\n\t\t\t--diff-algorithm= --default-prefix\n\t\t\t--submodule --submodule= --ignore-submodules\n\t\t\t--indent-heuristic --no-indent-heuristic\n\t\t\t--textconv --no-textconv --break-rewrites\n\t\t\t--patch --no-patch --cc --combined-all-paths\n\t\t\t--anchored= --compact-summary --ignore-matching-lines=\n\t\t\t--irreversible-delete --line-prefix --no-stat\n\t\t\t--output= --output-indicator-context=\n\t\t\t--output-indicator-new= --output-indicator-old=\n\t\t\t--ws-error-highlight=\n\t\t\t--pickaxe-all --pickaxe-regex\n'
declare -- __git_diff_difftool_options=$'--cached --staged\n\t\t\t--base --ours --theirs --no-index --merge-base\n\t\t\t--ita-invisible-in-index --ita-visible-in-index\n\t\t\t--stat --numstat --shortstat --summary\n\t\t\t--patch-with-stat --name-only --name-status --color\n\t\t\t--no-color --color-words --no-renames --check\n\t\t\t--color-moved --color-moved= --no-color-moved\n\t\t\t--color-moved-ws= --no-color-moved-ws\n\t\t\t--full-index --binary --abbrev --diff-filter=\n\t\t\t--find-copies --find-object --find-renames\n\t\t\t--no-relative --relative\n\t\t\t--find-copies-harder --ignore-cr-at-eol\n\t\t\t--text --ignore-space-at-eol --ignore-space-change\n\t\t\t--ignore-all-space --ignore-blank-lines --exit-code\n\t\t\t--quiet --ext-diff --no-ext-diff --unified=\n\t\t\t--no-prefix --src-prefix= --dst-prefix=\n\t\t\t--inter-hunk-context= --function-context\n\t\t\t--patience --histogram --minimal\n\t\t\t--raw --word-diff --word-diff-regex=\n\t\t\t--dirstat --dirstat= --dirstat-by-file\n\t\t\t--dirstat-by-file= --cumulative\n\t\t\t--diff-algorithm= --default-prefix\n\t\t\t--submodule --submodule= --ignore-submodules\n\t\t\t--indent-heuristic --no-indent-heuristic\n\t\t\t--textconv --no-textconv --break-rewrites\n\t\t\t--patch --no-patch --cc --combined-all-paths\n\t\t\t--anchored= --compact-summary --ignore-matching-lines=\n\t\t\t--irreversible-delete --line-prefix --no-stat\n\t\t\t--output= --output-indicator-context=\n\t\t\t--output-indicator-new= --output-indicator-old=\n\t\t\t--ws-error-highlight=\n\t\t\t--pickaxe-all --pickaxe-regex\n'
declare -- __git_format_patch_extra_options=$'\n\t--full-index --not --all --no-prefix --src-prefix=\n\t--dst-prefix= --notes\n'
declare -- __git_log_common_options=$'\n\t--not --all\n\t--branches --tags --remotes\n\t--first-parent --merges --no-merges\n\t--max-count=\n\t--max-age= --since= --after=\n\t--min-age= --until= --before=\n\t--min-parents= --max-parents=\n\t--no-min-parents --no-max-parents\n'
declare -- __git_log_gitk_options=$'\n\t--dense --sparse --full-history\n\t--simplify-merges --simplify-by-decoration\n\t--left-right --notes --no-notes\n'
declare -- __git_log_shortlog_options=$'\n\t--author= --committer= --grep=\n\t--all-match --invert-grep\n'
declare -- __git_log_show_options=$'\n\t--diff-merges --diff-merges= --no-diff-merges --dd --remerge-diff\n'
declare -- __git_merge_strategy_options=$'ours theirs subtree subtree= patience\n\thistogram diff-algorithm= ignore-space-change ignore-all-space\n\tignore-space-at-eol renormalize no-renormalize no-renames\n\tfind-renames find-renames= rename-threshold='
declare -- __git_mergetools_common=$'diffuse diffmerge ecmerge emerge kdiff3 meld opendiff\n\t\t\ttkdiff vimdiff nvimdiff gvimdiff xxdiff araxis p4merge\n\t\t\tbc codecompare smerge\n'
```

`COMP_WORDBREAKS` is the only one that uses escaped quotes on my system (although I assume other environmental variables in the wild will too).